vic_run/src/runoff.c

```diff
@@ -39,48 +39,51 @@ runoff(cell_data_struct  *cell,
        soil_con_struct   *soil_con,
        double             ppt,
        double            *frost_fract,
-       int                dt,
+       double             dt,
```

Comment:
Sure, after I finish initialize_atmos.c and the mtclim functions...

On Mon, Jan 12, 2015 at 10:23 PM, Bart Nijssen <notifications@github.com>
wrote:

> In vic_run/src/runoff.c
> <https://github.com/UW-Hydro/VIC/pull/188#discussion-diff-22843922>:
>
> > @@ -39,48 +39,51 @@ runoff(cell_data_struct  *cell,
> >         soil_con_struct   *soil_con,
> >         double             ppt,
> >         double            *frost_fract,
> > -       int                dt,
> > +       double             dt,
>
> @tbohn <https://github.com/tbohn> : Could you review runoff.c to make
> sure it does what it needs to. It had a hardwired hourly time loop before
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Hydro/VIC/pull/188/files#r22843922>.
>